### DISCUSSION TITLE
Any way to convert ogg files to akb.bytes without playing the sound in the game? #542

### DISCUSSION THREAD

**Discussion by Algarad** [Albeoris/Memoria#545] [Q&A]
Hi,

Having to reach the location and/or activate the condition for certain dialogues is a pain.
It would be great to be able to convert the ogg files to akb.bytes without having to play the game, perhaps with an external program.
A way to convert files in bulk, so to speech.

I can try generating the .ogg files directly with the AKB header. I haven't tried this way yet, I don't know how difficult it would be.

Thanks!

**Comment by SamsamTS** [comment]
I have a tool that does just that. Contact me on Discord and I'll share it to you.
